src/ResourceManager/Sql/Commands.Sql/Auditing/Model/BaseBlobAuditingPolicyModel.cs

```diff
@@ -18,8 +18,7 @@ public enum AuditActionGroups
     {
             BATCH_STARTED_GROUP,
             BATCH_COMPLETED_GROUP,
-            APPLICATION_ROLE_CHANGE_PASSWORD_GROUP, 
-            AUDIT_CHANGE_GROUP,
+            APPLICATION_ROLE_CHANGE_PASSWORD_GROUP,
```

Comment:
So, this is a breaking change, right?  You will need to deprecate these values, and, if provided by the user, map them into other values.  Otherwise, users with scripts that already use these values will be broken.

Reply:
@markcowl AUDIT_CHANGE_GROUP was never supported. This action group was mistakenly added, and when provided the backend throws an exception anyway.

Reply:
Also, it cannot be mapped to anything, it should just be removed, as it shouldn't have been there in the first place.

Reply:
@ranisha2 just to be 100% sure (☺️), if a user provided the `AUDIT_CHANGE_GROUP` value for any parameter that accepts an `AuditActionGroup` enum, then it would have _never_ worked and _always_ thrown an exception?

Reply:
@cormacpayne When enabling a policy with AUDIT_CHANGE_GROUP value, our backend will always throw an exception. The only way to provide this value with no exception to be thrown is when disabling the policy, which does not make any sense as this value does not take effect when the policy is disabled.

Reply:
@ranisha2 But could there be a customer script that used this value when disabling a policy?  As long as there can be no working script that uses this value, then this change is OK, otherwise, it is a breaking change, and the proper thing is to deprecate the value rather than removing it.  If possible, for enable scenarios, the value would be mapped to anohter (default) value, although throwing would be OK, if that is what the service does now.